### ISSUE TITLE
Monsters not spawning

### ISSUE THREAD

**Issue by NyarlathotepZERO** [itsdinkd/AQM2#127]
**Modpack Version**
1.01hf

**Minecraft Version**
1.19.2

**Describe the crash or bug**
Monsters not spawning

**Steps to reproduce the behavior:**
start new game, wander around in places you expect monsters

**Expected behavior**
Death

**Screenshots**
![image](https://user-images.githubusercontent.com/26475624/211113418-731c4d42-aa4e-4cf9-ad61-97ff12f92131.png)

**Crash log or latest.log.log**
none needed

**Additional context**
nothing to add

**Comment by NyarlathotepZERO** [comment]
In my single player game opened to LAN and enabled cheats. I then ran the '/kill @e[type=minecraft:skeleton]' as requested and it reported back 'killed 23 entities'.

I had a single mini zombie spawn on the surface that next morning which promptly burnt in the morning sunshine.

I then ran down a large cave following it for quite some time, I must have got close to Y-level zero and did not spot a single creature (save for a few bats). I came across a skeleton spawner which seemed to spawn skeletons fine who then (very) promptly disposed of me. 

In this whole adventure I would have expected to see many more monsters than the single mini zombie spawn.
